lib/fli/FliImpl.cpp

```diff
@@ -159,7 +330,36 @@ GpiObjHdl*  FliImpl::native_check_create(std::string &name, GpiObjHdl *parent)
  */
 GpiObjHdl*  FliImpl::native_check_create(uint32_t index, GpiObjHdl *parent)
 {
-    return NULL;
+    if (parent->get_type() == GPI_MODULE or parent->get_type() == GPI_ARRAY) {
+        char buff[15];
```

Comment:
@stuarthodgson Not really, just needed to be big enough to handle the largest unsigned int, i.e. 4294967295.  That will require 10 characters, plus 3 additional characters for "(", ")" and "\0".  I just preferred it to be a little larger.  If you want I can change it.